app/assets/stylesheets/footer.scss

```
@@ -68,6 +68,10 @@
   a {
     color: $white;
     text-decoration: none;
+    // Shrink the anchor to its text so the pointer cursor and hover
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)